plugins/modules/podman_play.py

```diff
@@ -244,6 +246,22 @@ def _command_run(self, cmd):
             self.module.log('PODMAN-PLAY-KUBE rc: %s' % rc)
         return rc, out, err
 
+    def tear_down_pods(self):
+        '''
+        Tear down the pod and contaiers by using --down option in kube play
+        which is supported since Podman 3.4.0
+        '''
+        changed = False
+        kube_file = self.module.params['kube_file']
+
+        rc, out, err = self._command_run([self.executable, "kube", "play", "--down", kube_file])
```

Comment:
We can probably check in followups that nothing is removed, or something is remove,a nd set `changed` var accordingly.
```
$ podman kube play --down /home/sshnaidm/tmp/lite-pod.yaml
Pods stopped:
17b422aa4a3f3f4563d4faf029d2556fb07890b225b097a7dfae4a3395914c3a
Pods removed:
17b422aa4a3f3f4563d4faf029d2556fb07890b225b097a7dfae4a3395914c3a
Secrets removed:
Volumes removed:

$ podman kube play --down /home/sshnaidm/tmp/lite-pod.yaml
Pods stopped:
Pods removed:
Secrets removed:
Volumes removed:

```